src/encoded/static/components/filegallery.js

```diff
@@ -528,24 +549,61 @@ class RawSequencingTable extends React.Component {
                                 file.biological_replicates[0] === partner.biological_replicates[0]) ||
                                 ((!file.biological_replicates || file.biological_replicates.length === 0) &&
                                 (!partner.biological_replicates || partner.biological_replicates.length === 0))) {
-                            // Both the file and its partner qualify as good pairs of each other. Let
-                            // them pass the filter, and record set their sort keys to the lower of
-                            // the two accessions -- that's how pairs will sort within a biological
-                            // replicate.
-                            partner.pairSortKey = file.title < partner.title ? file.title : partner.title;
-                            file.pairSortKey = partner.pairSortKey;
-                            file.pairSortKey += file.paired_end;
-                            partner.pairSortKey += partner.paired_end;
+                            // Both the file and its partner qualify as good pairs of each other.
+                            // Let hem pass the filter, and set their sort keys to the lower of the
```

Comment:
"them" and not "hem"

Reply:
I’m going to change that..

Reply:
did you change this?

Reply:
Yes. You’re looking at the outdated version.